lib/reviewable/actions.rb

```diff
@@ -55,6 +59,7 @@ def add_bundle(id, icon: nil, label: nil)
     end
 
     def add(id, bundle: nil)
+      id = [reviewable.target_type&.underscore, id].compact_blank.join("-")
```

Comment:
A reviewable queued post hasn't had the post record created yet, so `target_type` will be `nil`. Alternatively we can say that each `Reviewable` subclass needs to declare it's target type as well, but went with this for now.